packages/database-editor/src/components/ImportWizard/ImportWizard.tsx

```diff
@@ -1,20 +1,20 @@
 import type { DatabaseEditorContext } from '@axonivy/database-editor-protocol';
-import { BasicDialogHeader, Button, Dialog, DialogContent, DialogTrigger, toast } from '@axonivy/ui-components';
+import { BasicDialogHeader, Dialog, DialogContent, DialogTrigger, toast } from '@axonivy/ui-components';
+import { useState, type ReactNode } from 'react';
 import { useTranslation } from 'react-i18next';
 import './ImportWizard.css';
 import { WizardContent } from './WizardContent';
 
-export const ImportWizard = ({ context }: { context: DatabaseEditorContext }) => {
+export const ImportWizard = ({ context, children }: { context: DatabaseEditorContext; children: ReactNode }) => {
   const { t } = useTranslation();
+  const [open, setOpen] = useState<boolean>(false);
```

Comment:
It is not needed to define the generic type if it is clear from the init value
`  const [open, setOpen] = useState(false);`